samples/bluetooth/hci_uart/src/main.c

```diff
@@ -317,7 +317,7 @@ static int hci_uart_init(struct device *unused)
 	SYS_LOG_DBG("");
 
 	hci_uart_dev =
-		device_get_binding(CONFIG_BLUETOOTH_UART_TO_HOST_DEV_NAME);
+	device_get_binding(CONFIG_BLUETOOTH_CONTROLLER_TO_HOST_UART_DEV_NAME);
```

Comment:
yes, keep the orig indent. Or have the function call in the same line and have the parameter in the next line, indented by a tab.
hci_uart_dev = device_get_binding(
        CONFIG_B....);